.github/workflows/linux.yml

```diff
@@ -20,11 +20,12 @@ jobs:
       image: openquantumsafe/ci-ubuntu-jammy:latest
     env:
       MAKE_PARAMS: "-j 18"
+      LIBOQS_BRANCH="0.10.1"
```

Comment:
See below: this appears to be triggering a [yaml syntax error](https://github.com/open-quantum-safe/oqs-provider/actions/runs/9435507266/workflow) which has slipped through a crack in the GitHub UI.

I can push a fix to this branch if that is OK with you @baentsch.

Reply:
Absolutely (OK) -- Thanks, @SWilson4 !

Reply:
Also fixed this.